src/project-4.js

```diff
@@ -1,29 +1,48 @@
 const getFirstItem = (collection, cb) => {
+  cb(collection[0]);
   // invoke the callback function and pass the first item from the collection in as an argument
 };
 
 const getLength = (collection, cb) => {
+  cb(collection.length);
   // Write a function called getLength that passes the length of the array into the callback
 };
 
 const getLastItem = (collection, cb) => {
+  cb(collection[collection.length - 1]);
   // Write a function called getLastItem which passes the getLastItem item of the array into the callback
 };
 
 const sumNums = (x, y, cb) => {
+  cb(x + y);
   // Write a function called sumNums that adds two numbers and passes the result to the callback
 };
 
 const multiplyNums = (x, y, cb) => {
+  cb(x * y);
   // Write a function called multiplyNums that multiplies two numbers and passes the result to the callback
 };
 
 const contains = (collection, item, cb) => {
+  let isPresent = false;
+  for (let i = 0; i < collection.length; i++) {
```

Comment:
This looks really good. Shows that you know a lot. I'd look at ways to simplify this a little bit.